mlp/training/acceleration_dataset.py

```diff
@@ -170,8 +170,12 @@ def __init__(self, directory, test_directory=None, label_mapper=lambda x: x, add
             examples = self.load_examples(directory, label_mapper)
             examples.shuffle()
 
+            examples.print_statistic("all dataset", self.label_id_mapping)
+
             train, test = examples.split(self.TRAIN_RATIO)
-        
+
+        train.print_statistic("train", self.label_id_mapping)
+        test.print_statistic("test", self.label_id_mapping)
```

Comment:
Can this be moved to the training script? As this is not necessary for the notebooks.